boot/makebootable.go

```diff
@@ -78,6 +78,13 @@ func MakeBootableImage(model *asserts.Model, rootdir string, bootWith *BootableS
 	return makeBootable20(rootdir, bootWith, bootFlags)
 }
 
+// MakeBootablePartition configures a partition mounted on rootdir
+// using information from bootWith and bootFlags. Contrarily to
+// MakeBootableImage this happens in a live system.
+func MakeBootablePartition(partDir string, bootWith *BootableSet, bootMode string, opts *bootloader.Options, bootFlags []string) error {
```

Comment:
nitpick: maybe bootloader opts should go after partDir ?

Reply:
this needs a unit test

Reply:
Added now